<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowDialog</class>
 <widget class="QDialog" name="SlideShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
   <item>
    <widget class="QWidget" name="slideShow" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QWidget" name="slidePreviousWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="PictureButton" name="btnSlidePrevious">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PictureStateButton" name="btnPlay">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PictureButton" name="btnClose">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="PictureAnimationWidget" name="pictureAnimation" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="slideNextWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="PictureButton" name="btnSlideNext">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="previewShow" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <widget class="PreviewListWidget" name="previewListWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>16777215</width>
           <height>110</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureButton</class>
   <extends>QPushButton</extends>
   <header>picturebutton.h</header>
  </customwidget>
  <customwidget>
   <class>PictureStateButton</class>
   <extends>QPushButton</extends>
   <header>picturestatebutton.h</header>
  </customwidget>
  <customwidget>
   <class>PictureAnimationWidget</class>
   <extends>QWidget</extends>
   <header>pictureanimationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewListWidget</class>
   <extends>QListView</extends>
   <header>previewlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
